internal/go/skycfg/yaml_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018 The Skycfg Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package skycfg
+
+import (
+	"fmt"
+	"testing"
+
+	"go.starlark.net/starlark"
+)
+
+type TestCase struct {
```

Comment:
Duplicating this type between `json_test.go` and `yaml_test.go` is causing a build failure: https://travis-ci.com/stripe/skycfg/jobs/165113107
```
# github.com/stripe/skycfg/internal/go/skycfg [github.com/stripe/skycfg/internal/go/skycfg.test]
internal/go/skycfg/yaml_test.go:26:6: TestCase redeclared in this block
	previous declaration at internal/go/skycfg/json_test.go:26:6
FAIL	github.com/stripe/skycfg/internal/go/skycfg [build failed]
```

I recommend either naming the struct differently in each file (`JsonTestCase` and `YamlTestCase`), or deleting one of the definitions.

Reply:
well, not exporting didn't work, so I have followed your suggestion, except that the linter was complaining about `JsonTestCase` name and forced me to use `JSONTestCase` instead :)